### ISSUE TITLE
Small links are difficult to tap

### ISSUE THREAD

**Issue by Terrance** [foobnix/LibreraReader#1339]
Links to footnotes (using the likes of *, †, ¹, → etc. as the link text), as well as any other links of single or few characters, are difficult to accurately tap without taking several attempts.  If I've tweaked the zoom/positioning of the page by pinch-zooming, attempting to tap links too fast also results in a double-tap of the page being interpreted as a position reset, meaning I have to resize the page again.

It would be useful to have some kind of tap zone padding around links on the page so that you don't need to be as accurate to tap them (the page layout does not need to change; it's rare that two links end up next to each other).  Alternatively, a way to pull up a menu/list of links in the visible page could also work, similar to the Bookmarks and Contents popups.

**Comment by Korb** [comment]
I agree. In cases where footnotes are formatted as a number in uppercase, I can't click on the link even after the tenth time, and eventually I simply stop using footnotes.
